backend/src/main/java/gov/cdc/usds/simplereport/service/LoggedInAuthorizationService.java

```diff
@@ -56,15 +56,9 @@ public List<OrganizationRoleClaims> findAllOrganizationRoles() {
     List<OrganizationRoleClaims> oktaOrgRoleClaims =
         _extractor.convert(currentAuth.getAuthorities());
 
-    if (!isSiteAdmin()) {
+    if (!isSiteAdmin() && _featureFlagsConfig.isOktaMigrationEnabled()) {
       String username = currentAuth.getName();
-      List<OrganizationRoleClaims> dbOrgRoleClaims =
```

Comment:
⚠️ Before I was doing the comparison check way lower in the stack for `whoami` calls. 

Moved the check to the level of the ` getCurrentUserInfoForWhoAmI` method to avoid instances where users who were not migrated were being checked (thus, logging an error).